packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -74,7 +99,63 @@ export function DatePicker({
                     </div>
                     <table data-testid="jkl-datepicker-calendar" />
                 </CoreDatepicker>
-            </CoreToggle>
+            </div>
+            <SupportLabel errorLabel={errorLabel} helpLabel={helpLabel} />
         </div>
     );
 }
+
+function useOnClickOutside<T extends HTMLElement>(onClickOutside: () => void): MutableRefObject<T | null> {
+    const ref = useRef<T>(null);
+
+    function onMouseDown(e: MouseEvent | FocusEvent) {
+        if (ref.current && !ref.current.contains(e.target as Node)) {
+            onClickOutside();
+        }
+    }
+
+    useEffect(() => {
+        // Bind the event listener
+        document.addEventListener("mousedown", onMouseDown);
+        // @ts-ignore
+        document.addEventListener("focusin", onMouseDown);
```

Comment:
`@ts-ignore` is unfortunately required, as the `focusin` event is not yet registered in the `FocusEvent` interface. The event type is supported by all browsers we should consider, and should be patched into TypeScript soon (see https://github.com/Microsoft/TypeScript/issues/30716).

Reply:
@piofinn 
I think we should add a reference to that issue in the // @ts-ignore. Otherwise - that comment will never be fixed.